scripts/node.sh

```diff
@@ -123,6 +123,7 @@ options:
    -y             run in legacy, foundational-node mode (default)
    -M             support multi-key mode (default: off)
    -A             enable archival node mode (default: off)
+   -B blacklist   specify file containing blacklisted accounts as a newline delimited file (default: ./blacklist.txt)
```

Comment:
default in .hmy directory would be better.

Reply:
will move